src/app/containers/Video/index.jsx

```diff
@@ -43,16 +47,29 @@ const VideoContainer = ({ blocks }) => {
     ['blocks', 1, 'model', 'blocks', 0, 'model', 'locator'],
     aresMediaBlock.model,
   );
-  const items = [
-    {
-      versionID,
-      kind,
-      duration,
+  const env = process.env.APP_ENV;
+  const mediaPlayerSettings = {
+    product: 'news',
+    responsive: true,
+    statsObject: { clipPID: pid },
+    playlistObject: {
```

Comment:
I'd expect this object to contain either a `guidance` or `warning` property.

Reply:
@oluoluoxenfree Thanks for making this change. Still not seeing any guidance warnings however?